src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -59,11 +59,13 @@ public Optional<EntityType> getEntityTypeDefinition(UUID entityTypeId) {
       .getEntityTypeDefinition(entityTypeId)
       .map((EntityType entityType) -> {
         entityType.setLabelAlias(localizationService.getEntityTypeLabel(entityType.getName()));
-
-        entityType.getColumns().forEach(column -> 
-          column.setLabelAlias(localizationService.getEntityTypeColumnLabel(entityType.getName(), column.getName()))
-        );
-
+        entityType.getColumns().forEach(column -> {
+          if (Boolean.TRUE.equals(column.getIsCustomField())) {
+            column.setLabelAlias(column.getName());
+          } else {
```

Comment:
Not a huge deal, but it might be worthwhile to add a short comment here explaining why we're not localizing custom field names (because they are user-defined at runtime, so we have no translations available)